test/openal_playback.cpp

```diff
@@ -257,6 +257,11 @@ int main() {
 #elif defined(TEST_ANIMATED_LOOPED_DOPPLER_PLAYBACK)
   printf("You should hear a continuously looping clip of the 1902 piano song \"The Entertainer\" played back at a dynamic playback rate that smoothly varies its pitch according to a sine wave doppler shift. Press OK when confirmed.\n");
 #elif defined(TEST_ANIMATED_LOOPED_PANNED_PLAYBACK)
+  assert(!alIsEnabled(AL_SOURCE_DISTANCE_MODEL));
+  alDisable(AL_SOURCE_DISTANCE_MODEL);
+  assert(!alIsEnabled(AL_SOURCE_DISTANCE_MODEL));
+  alEnable(AL_SOURCE_DISTANCE_MODEL);
+  assert(alIsEnabled(AL_SOURCE_DISTANCE_MODEL));
```

Comment:
Is there a reason you put this in the `TEST_ANIMATED_LOOPED_PANNED_PLAYBACK` block?  (Rather than up on line 247?)

Reply:
It seems that the block around 247 is a common one and I thought I was supposed to 

> update one of the existing tests

:)
Additionally I thought it might be a good idea to test if the option being enabled does not mess up the sound spatialization, while keeping the default for all the other tests.

Reply:
Fair enough.  LGTM.